pages/embed/contribution-flow.js

```diff
@@ -149,6 +153,9 @@ class NewContributionFlowPage extends React.Component {
     }
 
     const contributionBLocker = getContributionBlocker(LoggedInUser, account, tier, Boolean(this.props.tierId));
```

Comment:
Minor typo in the naming; 

```suggestion
    const contributionBlocker = getContributionBlocker(LoggedInUser, account, tier, Boolean(this.props.tierId));
```

Reply:
Hey @SudharakaP,  I think it was done purposefully to differentiate it from `ContributionBlocker` component

Reply:
@saideepesh000 : I don't think so. `ContributionBlocker` starts with uppercase `C` anyways. This seems more like a typo. 😄